wp-content/plugins/zippicks-geo-service/admin/class-admin.php

```
@@ -0,0 +1,774 @@
+                submit_button();
+                ?>
+            </form>
+        </div>
+        <?php
+    }
+    
+    /**
+     * Display statistics page
+     */
+    public function display_stats_page() {
+        global $wpdb;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$wpdb'. (Unused Code Rules)

(UnusedLocalVariable)